lib/modules/manager/gitlabci/utils.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { getGitlabDep } from './utils';
+
+describe('modules/manager/gitlabci/utils', () => {
+  describe('getGitlabDep', () => {
+    it.each([
```

Comment:
can we use the string template table syntax instead, it's much more readable. checkout versioning tests for samples.

Reply:
@viceice could please have another look at this PR. Tests use now string template table syntax.

Reply:
Please 👉 https://github.com/renovatebot/renovate/blob/main/.github/contributing.md#re-requesting-a-review